python/bench/ellipse_overlap_H.py

```diff
@@ -57,6 +57,18 @@ def frame2ellipse(fa):
 
 
 def ellipse_warp(H, ell_o, method):
+    """
+    TODO: Xu, could you add a description here?
+        Can we also add inline comments?
+
+        Args:
+        -   H:
+        -   ell_o:
+        -   method:
+
+        Returns:
+        -   well
+    """
     ell = copy.copy(ell_o)
```

Comment:
Hey @spongezhang, we're trying to make sure we're on the same page for all of these files. Do you think you could add inline comments to these functions and also a description in the function signature with input/output argument types? We can mark the ones we think we most need if that works for you. Thanks so much!

Reply:
This piece of code is a bit tough. It comes from the old vlbenchmark. I may need some time to read the understand it.

Reply:
@spongezhang Got it -- no worries. Looks like Karel's original code was here: https://github.com/lenck/vlb/blob/master/matlab/%2Butls/ellipse_warp.m. Would be awesome if you could figure out why the warp is carried out this way -- any chance you could ping Karel about that?

Reply:
Hi John,

The code is dated back to some very old codes in vlbenchmark. I believe we can find all the theory in "A Comparison of Affine Region Detectors". 

Some codes in the benchmark is not easy to understand, especially vgg_compute_ellipse_overlap.pyx. Generally, you don't need to care too much about it. 

Best,
Xu